evalml/tests/pipeline_tests/test_component_graph.py

```diff
@@ -417,7 +367,7 @@ def test_get_estimators(example_graph):
         LogisticRegressionClassifier(),
     ]
 
-    component_graph = ComponentGraph.from_list(["Imputer", "One Hot Encoder"])
+    component_graph = ComponentGraph({"Imputer": ["Imputer", "X", "y"]})
```

Comment:
I thought this test still had validity: it was testing that `get_estimators` when a component graph has no estimators will return an empty list. I'm just changing how the graph is initialized :)